Library/Homebrew/cmd/test-bot.rb

```diff
@@ -279,6 +279,7 @@ def diff_formulae(start_revision, end_revision, path, filter)
           "diff-tree", "-r", "--name-only", "--diff-filter=#{filter}",
           start_revision, end_revision, "--", path
         ).lines.map do |line|
+          next unless @tap
```

Comment:
Shouldn't this be put in the first line of this method instead of inside a loop.